examples/rollup/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Build a full, working integration using Connect embedded UIs. Included are some basic build and run scripts you can use to start up the application.",
   "main": "index.js",
   "dependencies": {
-    "@stripe/connect-js": "file:stripe-connect-js-1.0.0.tgz"
+    "@stripe/connect-js": ">=2.0.1"
```

Comment:
Shouldn't this be targeting the local via yalc?

Reply:
This was originally targeting local because the package wasn't published yet, but now that it is we should use the published version. This example app is to allow users to run this library from the same repo. Stripe-JS also follows the same example-https://github.com/stripe/stripe-js/blob/master/examples/rollup/package.json

Reply:
:portalto: https://github.com/stripe/connect-js/pull/54/files#r1355777046